entrypoint.sh

```diff
@@ -46,7 +46,7 @@ appendParams "$INPUT_SERVICE_NAME"
 appendParams $namespace_arg
 
 case $INPUT_SERVICE_OPERATION in
-  create | update )
+  create | update | apply )
```

Comment:
should `delete` be allowed?

Reply:
I have removed this from action, because I don't see a use case here.
If there is usecase, I will have to handle the `delete` seperately.

Reply:
I think there is a use case for delete.

Reply:
Okay, if there is. I will add.

Reply:
In general I think any resources that an action (or any program) can create, it should also be able to remove

Reply:
Okay :+1: